views/tables/AdminUsersTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import Settings from '@mui/icons-material/Settings'
+import Clear from '@mui/icons-material/Clear'
+import LockOpen from '@mui/icons-material/LockOpen'
+import IconButton from '@mui/material/IconButton'
+import Checkbox from '@mui/material/Checkbox'
+import { DatePicker } from '@mui/x-date-pickers/DatePicker'
+import moment from 'moment'
+
+import Table from '../Table'
+
+const AdminUsersTable = (props) => {
+  const headers = [
+    {
+      dataProperty: 'uid',
+      label: 'Username',
+      sortable: true,
+    },
+    {
+      dataProperty: 'display_name',
+      label: 'Name',
+      sortable: true,
+    },
+    {
+      dataProperty: 'mail',
+      label: 'Email',
+      sortable: true,
+    },
+    {
+      dataProperty: 'role',
+      label: 'Role',
+      sortable: true,
+    },
+    {
+      dataProperty: 'access',
+      label: 'Access',
+      sortable: false,
+    },
+    {
+      dataProperty: 'account_expires',
+      label: 'Account Expiration',
+      sortable: true,
+    },
+    {
+      dataProperty: 'force_reset_pw',
+      label: 'Reset Password',
+      sortable: false,
+    },
+    {
+      dataProperty: 'blocked',
+      label: 'Inactive',
+      sortable: true,
+    },
+    {
+      dataProperty: 'delete',
+      label: 'Delete',
+      sortable: false,
+    }
+  ]
+
+  const cellRenderer = (user, property, rowIndex) => {
```

Comment:
Should we use the controlled components for the form values..?

Reply:
Eventually sure, but we'd have to totally rewrite a lot of the page logic.